src/bindgen/ir/enumeration.rs

```diff
@@ -90,6 +90,15 @@ impl Enum {
     }
 
     pub fn rename_fields(&mut self, config: &Config) {
+        if config.enumeration.prefix_with_enum_name ||
+           self.annotations.bool("prefix_with_enum_name").unwrap_or(false)
```

Comment:
Could you change the annotation to `prefix-with-name`? The other annotations use dashes and the enum part can be inferred, I think. The config value should be `prefix_with_name` also if you do this.